### PR TITLE
fix: moves `ln` to `postinst` and related changes

### PR DESCRIPTION
**Pull request by wizard-28**
Fixes all issues for pacscripts raised in #1599. Except for `ardour-git` and `wine-staging-git`

### CODE REVIEW

File: packages/awesome-git/awesome-git.pacscript [hunk 1]
```diff
@@ -11,8 +11,8 @@ pkgname="awesome"
 pkgdir="/usr/src/pacstall/${name}"
 description="awesome window manager"
 url="https://github.com/awesomeWM/${pkgname}.git"
-depends="libdbus-1-dev libxcb-xfixes0-dev libxcb-randr0-dev libxcb-xinerama0-dev libxcb-icccm4-dev libxcb-xtest0-dev liblua5.3-dev libcairo2-dev dbus libgdk-pixbuf2.0-dev libxdg-basedir-dev libxkbcommon-x11-dev lua5.3 lua-lgi-dev libpango1.0-dev libstartup-notification0-dev libxcb-cursor-dev libxcb-keysyms1-dev libxcb-util-dev libxcb-xrm-dev libxfixes-dev cmake"
-build_depends="asciidoc docbook-xsl git imagemagick lua-ldoc xmlto"
+depends="libdbus-1-dev libxcb-xfixes0-dev libxcb-randr0-dev libxcb-xinerama0-dev libxcb-icccm4-dev libxcb-xtest0-dev liblua5.3-dev libcairo2-dev dbus libgdk-pixbuf2.0-dev libxdg-basedir-dev libxkbcommon-x11-dev lua5.3 lua-lgi-dev libpango1.0-dev libstartup-notification0-dev libxcb-cursor-dev libxcb-keysyms1-dev libxcb-util-dev libxcb-xrm-dev libxfixes-dev"
+build_depends="asciidoc cmake docbook-xsl git imagemagick lua-ldoc xmlto"
```
Comment (Henryws) [inline packages/awesome-git/awesome-git.pacscript:15]: Most of the `depends` are actually `build_depends`.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/awesome-git/awesome-git.pacscript (hunk 1 above)
Comment (wizard-28) [reply packages/awesome-git/awesome-git.pacscript:15]: Suggest the change.

Comment (Henryws) [inline packages/awesome-git/awesome-git.pacscript:15]:
```suggestion
depends="libx11-6 libxcb-cursor0 libxcb-icccm4 libxcb-keysyms1 libxcb-randr0 libxcb-shape0 libxcb-util1 libxcb-xinerama0 libxcb-xkb1 libxcb-xrm0 libxcb-xtest0 libxcb1 libxdg-basedir1 libxkbcommon-x11-0 libxkbcommon0 libstartup-notification0 libglib2.0-0 libgdk-pixbuf2.0-0 libgdk-pixbuf-2.0-0 libdbus-1-3 libcairo2 libcairo-gobject2 gir1.2-glib-2.0 gir1.2-gdkpixbuf-2.0 gir1.2-freedesktop lua5.3 liblua5.3-0 dbus-x11 libnotify-bin wmctrl x11-apps xcb-proto xdotool xorg lua-lgi"
build_depends="asciidoctor gir1.2-pango-1.0 gir1.2-gtk-3.0 git gzip lua-ldoc imagemagick asciidoc xmlto cmake lua-busted lua-check liblua5.3-dev libxcb-cursor-dev libxcb-util0-dev libxcb-keysyms1-dev libxcb-icccm4-dev libxcb-xkb-dev libxkbcommon-dev libstartup-notification0-dev libxdg-basedir-dev libxcb-xrm-dev libxkbcommon-x11-dev lua-lgi libglib2.0-dev libgdk-pixbuf2.0-dev libxcb-xinerama0-dev gettext libdbus-1-dev libgirepository1.0-dev libpango1.0-dev libx11-xcb-dev libxcb-randr0-dev libxcb-shape0-dev libxcb-xfixes0-dev xutils-dev libluajit-5.1-dev libcairo2-dev lua-discount x11proto-dev libxcb-xtest0-dev libmarkdown2 libyaml-dev lua-any lua-cliargs lua-dkjson lua-filesystem lua-inifile lua-luassert lua-mediator lua-penlight lua-say lua-system lua-term lua5.2 libc6 luajit"
```

Taken from the MPR.

File: packages/gfn-electron-bin/gfn-electron-bin.pacscript [hunk 2]
```diff
@@ -32,6 +32,12 @@ install() {
...
     sudo chmod +x "${STOWDIR}/${name}/opt/${pkgname}/geforcenow-electron"
+}
+
+postinst() {
+    sudo ln -s -r "/opt/${pkgname}/geforcenow-electron" "/usr/bin/geforcenow"
+}
 
-    sudo ln -s -r "${STOWDIR}/${name}/opt/${pkgname}/geforcenow-electron" "${STOWDIR}/${name}/usr/bin/geforcenow"
+removescript() {
+    sudo rm "/usr/bin/geforcenow"
```
Comment (Henryws) [inline packages/gfn-electron-bin/gfn-electron-bin.pacscript:42]:
```suggestion
    sudo rm -f "/usr/bin/geforcenow"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — COMMENTED** [review]
i3-gaps doesn't work, but that's not relating to the symlink changes. So fix that later @wizard-28

File: packages/onnxruntime-1-7-0-bin/onnxruntime-1-7-0-bin.pacscript [hunk 3]
```diff
@@ -15,11 +15,15 @@ build() {
...
-  sudo rm "lib${pkgname}.so" 2> /dev/null
-  sudo ln -s "./lib${pkgname}.so.${version}" "lib${pkgname}.so"
-  cd - > /dev/null
-  rm lib/*
   sudo install -Dm644 include/* -t "$STOWDIR/$name/usr/local/include"
   sudo install -Dm644 LICENSE -t "$STOWDIR/$name/usr/share/licenses/$name"
 }
+
+postinst() {
+  sudo rm "/usr/local/lib/lib${pkgname}.so" 2> /dev/null
```
Comment (Henryws) [inline packages/onnxruntime-1-7-0-bin/onnxruntime-1-7-0-bin.pacscript:23]:
```suggestion
  sudo rm -f "/usr/local/lib/lib${pkgname}.so" 2> /dev/null
```

File: packages/onnxruntime-1-7-0-bin/onnxruntime-1-7-0-bin.pacscript [hunk 4]
```diff
@@ -15,11 +15,15 @@ build() {
...
+  sudo ln -s "/usr/local/lib/lib${pkgname}.so.${version}" "/usr/local/lib/lib${pkgname}.so"
```
Comment (Henryws) [inline packages/onnxruntime-1-7-0-bin/onnxruntime-1-7-0-bin.pacscript:24]:
```suggestion
  sudo ln -sf "/usr/local/lib/lib${pkgname}.so.${version}" "/usr/local/lib/lib${pkgname}.so"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/yabridge/yabridge.pacscript [hunk 5]
```diff
@@ -7,33 +7,21 @@ hash="e9b66d48339cec5f3cc929d1da37d73c78c04d98281d051a2cbbe20dfb8f9eba"
...
+  sudo install yabridge-host{,-32}.exe{,.so} "$pkgdir/usr/bin"
 
-removescript() {
-  sudo rm "/usr/bin/yabridgectl"
-  sudo rm "/usr/bin/yabridge-host.exe"
-  sudo rm "/usr/bin/yabridge-host.exe.so"
-  sudo rm "/usr/bin/yabridge-host-32.exe"
-  sudo rm "/usr/bin/yabridge-host-32.exe.so"
+  sudo install -dm755 "$pkgdir/usr/lib"
+  sudo install libyabridge{,-chainloader}-{vst2,vst3}.so "$pkgdir/usr/lib"
```
Comment (Henryws) [inline packages/yabridge/yabridge.pacscript:26]: This doesn't make sense. `$pkgdir` doesn't exist.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/yabridge/yabridge.pacscript (hunk 5 above)
Comment (Henryws) [reply packages/yabridge/yabridge.pacscript:26]: Wait nvm I read it wrong

**Review by Henryws — APPROVED** [review]

**Pull request merged**